public/index.html

```diff
@@ -1,10 +1,79 @@
-<!DOCTYPE html>
-<html>
-	<head>
-    <title>Create a Web page</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">   
-	</head>
-<body>
-	<h1>Here you go!</h1>
-</body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" type="text/css" href="index_style.css">

```

Comment:
@rupamverma Is there any reason for adding all kinds of css (internal, external and inline css)